app/controllers/projects_controller.rb

```diff
@@ -72,7 +82,16 @@ def destroy
   end
   
   private def find_section
-    @section = Section.find(params[:section_id])
+    if is_instructor_html
```

Comment:
Shouldn't this check of the instructor being in the section happen at every call on projects? For instance, edit, update, remove, and destroy don't check if the instructor is in the section. Maybe a check that happens before every call that checks if its an instructor which are in the section.